lib/_internal/_a2p/index.js

```diff
@@ -1,5 +1,3 @@
-export default function a2p(a) {
-  return Array.isArray(a)
-    ? a.join('.')
-    : typeof a === 'string' || typeof a === 'number' ? '' + a : ''
-}
+export default (a) => Array.isArray(a)
```

Comment:
With often used static methods imported from separate module, in this case `export const { isArray } = Array`, the whole thing should minify even better.

Reply:
You're completely right! If only @Beraliv likes that, I'd love to implement it!

Reply:
> With often used static methods imported from separate module, in this case `export const { isArray } = Array`, the whole thing should minify even better.

@plesiecki This is good for functions which have the same number of arguments as our implementation (and the same order)

> You're completely right! If only @Beraliv likes that, I'd love to implement it!

@erykpiast so yeah, you should change that for sure! 🚀